cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -607,4 +610,76 @@ public ClientBuilder readTimeout(long readTimeout, TimeUnit readTimeoutUnit) {
         return this;
     }
 
+    /**
+     * Sets Cloudant client credentials by inspecting the VCAP_SERVICES Cloud Foundry environment
+     * variable. The VCAP_SERVICES variable contains connection information to access a Bluemix
+     * service instance.
+     * <P>
+     * Note: Only a single Cloudant service must exist in VCAP_SERVICES.
```

Comment:
I think we should explicitly point users at the alternative method. Maybe append to the note something like:
> `If there are multiple bound Cloudant service instances then use {@link #bluemix(String)} to select an instance by name.`